src/Dnsimple/Client.php

```diff
@@ -81,6 +82,10 @@ class Client
      * @var Accounts The service handling the Accounts API
      */
     public $accounts;
+    /**
+     * @var Billing The service handling the Billing API
+     */
+    public $billing;
```

Comment:
This needs to be registered at the bottom of the file.

Reply:
@AGS4NO this has not yet been resolved. The change needs to be made to https://github.com/dnsimple/dnsimple-php/blob/4c48d82ba25bffc475e1264a0bb48ef443149379/src/Dnsimple/Client.php#L287C16-L287C16